components/script/dom/intersectionobserverrootmargin.rs

```diff
@@ -3,14 +3,16 @@
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 
 //! Copy of Stylo Gecko's [`style::values::specified::gecko::IntersectionObserverRootMargin`] implementation.
-//! TODO: expose the object to servo as well in Stylo
+//! TODO(stevennovaryo): when stylo's IntersectionObserverMargin is up, make a thin wrapper and remove copied codes
```

Comment:
If you mean https://phabricator.services.mozilla.com/D237514, it's already available

Reply:
Updated, Thanks!